packages/amplify-provider-awscloudformation/src/aws-utils/aws-pinpoint.js

```diff
@@ -43,11 +43,13 @@ async function getConfiguredPinpointClient(context, category, action, options =
   };
 
   if (httpProxy) {
-    aws.config.update({
+    awsConfig = {
+      ...awsConfig,
       httpOptions: {
         agent: proxyAgent(httpProxy),
+        rejectUnauthorized: context.ignoreSSL,
```

Comment:
the rejectUnauthorized field should be in the agent, instead of the httpOptions

Reply:
Unless I'm misreading the API-Docs, there's no option for that in the agent constructor. You can only set them in the options themselves?

https://nodejs.org/api/https.html#https_https_request_url_options_callback